web/less/landing.less

```diff
@@ -95,7 +95,7 @@ body {
     label {
       color: #b6bfc8;
       float: right;
-      width: 85px;
+      width: 90px;
```

Comment:
I made another pr for the form stuff but I didn't copy this over as not sure what it is.